docs/specs/pn.swagger.yaml

```diff
@@ -252,6 +252,12 @@ definitions:
       deviceId:
         type: string
         description: a unique id for that user/device pair
+      sandbox:
+        type: boolean
+        default: false
```

Comment:
Please add a description when this should be set, and what happens when you set it.